specification/cdn/resource-manager/readme.md

```diff
@@ -28,20 +28,31 @@ These are the global settings for the Cdn API.
 title: CdnManagementClient
 description: Cdn Management Client
 openapi-type: arm
-tag: package-2021-06
+tag: package-preview-2022-05
```

Comment:
@jianyexi, do you know what `tag` means here? Is it the default tag that should be used? https://github.com/Azure/azure-rest-api-specs/issues/20652